src/plugins/vis_types/timeseries/server/lib/vis_data/request_processors/table/date_histogram.ts

```diff
@@ -24,7 +24,8 @@ export const dateHistogram: TableRequestProcessorsFunction =
 
     const meta: TableSearchRequestMeta = {
       timeField,
-      index: panel.use_kibana_indexes ? seriesIndex.indexPattern?.id : undefined,
```

Comment:
nit: In `src/plugins/vis_types/timeseries/server/lib/vis_data/request_processors/annotations/query.test.ts` and `src/plugins/vis_types/timeseries/server/lib/vis_data/request_processors/table/query.test.ts` there are a few leftovers of the `index` property.

Reply:
@flash1293 these indexes related to filter object. It's not related to that PR

<img width="797" alt="image" src="https://user-images.githubusercontent.com/20072247/156349606-7e4587dc-2eee-4bfe-b685-bbb55ed72a69.png">